tmt/steps/prepare/shell.py

```diff
@@ -29,6 +32,23 @@ class PrepareShellData(tmt.steps.prepare.PrepareStepData):
         unserialize=lambda serialized: [ShellScript(script) for script in serialized],
     )
 
+    url: Optional[str] = field(
+        default=None,
+        option='--url',
+        metavar='REPOSITORY',
+        help='Git repository URL for fetching shell scripts.',
+    )
+
+    ref: Optional[str] = field(
+        default=None,
+        option='--ref',
+        metavar='REVISION',
+        help="""
+            Branch, tag or commit specifying the desired git
```

Comment:
How about "Branch, tag or commit to check out in the git repository cloned when ``url`` is specified." ?

Reply:
Updated !